packages/pangea-sdk/.sdk-ci.yml

```diff
@@ -35,6 +35,13 @@ generate_docs:
     paths: ['packages/pangea-sdk/python_sdk.json']
     when: on_success
 
+mypy:
```

Comment:
nit: I'd call this job `type_check` and the one above `lint`

Reply:
Done.